Cargo.toml

```diff
@@ -27,7 +27,7 @@ documentation = "https://sawtooth.hyperledger.org/docs/core/releases/latest/"
 edition = "2018"
 
 [features]
-default = ["pem"]
+default = ["pem", "log-zmq-stream"]
```

Comment:
As far as I know if you wish to disable these log messages then you will have to recompile whichever binary uses the SDK. Not really very convenient. 

Is it possible to propagate this into command line arguments ?

Reply:
Yes, using `fern::Dispatch`.